dev-packages/e2e-tests/test-applications/node-connect-app/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
```

Comment:
Since we are only typescript to run the app with ts-node and to check types we can probably configure this to not emit any files and then we can also get rid of the gitignore.